### PR TITLE
fuse: update distfiles; moved to github.

### PR DESCRIPTION
**Pull request by stpx**

### CODE REVIEW

File: srcpkgs/fuse-sshfs/template [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
 depends="openssh"
 short_desc="FUSE client based on the SSH File Transfer Protocol"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
-homepage="http://fuse.sourceforge.net/sshfs.html"
 license="GPL-2"
-distfiles="${SOURCEFORGE_SITE}/fuse/sshfs-fuse-$version.tar.gz"
-checksum=e9171452e5d0150b9c6a2158fd2e2dcefb5d5d03ba4d208949e00a3a46c6e63e
+homepage="https://github.com/libfuse/sshfs"
+distfiles="${homepage}/archive/sshfs_${version//./_}.tar.gz"
+checksum=f21321a619099c0957c61747c05f1a82359f5ccf5a8bfc75c519623571837c90
```
Comment (Gottox) [inline srcpkgs/fuse-sshfs/template]: I'd like to see a diff of the old and the new package.

File: srcpkgs/fuse/template [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
+hostmakedepends="automake libtool gettext-devel"
 short_desc="Filesystem in Userspace"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2"
-homepage="http://fuse.sourceforge.net"
-distfiles="${SOURCEFORGE_SITE}/$pkgname/$pkgname-$version.tar.gz"
-checksum=6be9c0bff6af8c677414935f31699ea5a7f8f5f791cfa5205be02ea186b97ce1
+homepage="https://github.com/libfuse/libfuse"
+distfiles="${homepage}/archive/${pkgname}_${version//./_}.tar.gz"
+checksum=23d4696c7d1cbaa91a02324480fb500312fbe940515636b96024b070cdc06073
```
Comment (Gottox) [inline srcpkgs/fuse/template]: Same here.

File: srcpkgs/python-fuse/template [hunk 3]
```diff
@@ -10,6 +9,6 @@ pycompile_module="fuse"
 short_desc="FUSE bindings for Python2"
 maintainer="Eivind Uggedal <eivind@uggedal.com>"
 license="LGPL-2.1"
-homepage="http://sourceforge.net/projects/fuse/"
-distfiles="${SOURCEFORGE_SITE}/fuse/fuse-python-${version}.tar.gz"
-checksum=11ead462f8d0f8302ee07b33fe93dc42c653861a16c8802a31903a7355b8351b
+homepage="https://github.com/libfuse/python-fuse"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=9db4eaa2ab8b8767ad1137981d9cf4b838c0425482e89e50e299799b7a47b89b
```
Comment (Gottox) [inline srcpkgs/python-fuse/template]: Same here.

File: srcpkgs/fuse/template (hunk 2 above)
Comment (stpx) [inline srcpkgs/fuse/template]: `fuse`:
```diff
diff -Nur ./fuse-2.9.4_3/files.plist fuse-2.9.4_4/files.plist
--- ./fuse-2.9.4_3/files.plist	1969-12-31 19:00:00.000000000 -0500
+++ fuse-2.9.4_4/files.plist	1969-12-31 19:00:00.000000000 -0500
@@ -39,7 +39,7 @@
 			<key>file</key>
 			<string>/usr/bin/fusermount</string>
 			<key>mtime</key>
-			<integer>1451698762</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
 			<string>499d7f40df6a30c10fffdcb439f4c956acffa16ff0a62ab294ae8e30eacc4bc8</string>
 		</dict>
@@ -47,7 +47,7 @@
 			<key>file</key>
 			<string>/usr/bin/ulockmgr_server</string>
 			<key>mtime</key>
-			<integer>1451698762</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
 			<string>2e9d8c88ef1fc43fa60d6219cfebeb56f746f026d0797628e76ff8c6b7316c17</string>
 		</dict>
@@ -55,7 +55,7 @@
 			<key>file</key>
 			<string>/usr/bin/mount.fuse</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
 			<string>5f631effc489ae71fe5491d4440e5d58a5741caef267fbb35dc628f8571749d6</string>
 		</dict>
@@ -63,7 +63,7 @@
 			<key>file</key>
 			<string>/usr/share/man/man8/mount.fuse.8</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>61c792e4007c067e692a34a5474fc4a2ee05dba003e4ff1db331a34206208807</string>
 		</dict>
@@ -71,7 +71,7 @@
 			<key>file</key>
 			<string>/usr/share/man/man1/ulockmgr_server.1</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>6a96152c2570c70b75c2b455e9252efb54e941a5b276c519c3fdf2b717579f73</string>
 		</dict>
@@ -79,7 +79,7 @@
 			<key>file</key>
 			<string>/usr/share/man/man1/fusermount.1</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>86bcfbd7914054addda1553f4815740ba0daaa0ee2a9deb8081a521d95f5dea6</string>
 		</dict>
@@ -87,7 +87,7 @@
 			<key>file</key>
 			<string>/usr/lib/libulockmgr.so.1.0.1</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
 			<string>f3ab8a3257473b60fab27142235c6281e2ea6b5b37f0f3c311e8be46f3e53257</string>
 		</dict>
@@ -95,7 +95,7 @@
 			<key>file</key>
 			<string>/usr/lib/libfuse.so.2.9.4</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
 			<string>79828b07beb71b4568b77284e0844e50214b150797c97a86fd2baae1bd6bcce6</string>
 		</dict>
@@ -106,7 +106,7 @@
 			<key>file</key>
 			<string>/usr/lib/libfuse.so.2</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>target</key>
 			<string>/usr/lib/libfuse.so.2.9.4</string>
 		</dict>
@@ -114,7 +114,7 @@
 			<key>file</key>
 			<string>/usr/lib/libulockmgr.so.1</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>target</key>
 			<string>/usr/lib/libulockmgr.so.1.0.1</string>
 		</dict>
diff -Nur ./fuse-2.9.4_3/props.plist fuse-2.9.4_4/props.plist
--- ./fuse-2.9.4_3/props.plist	1969-12-31 19:00:00.000000000 -0500
+++ fuse-2.9.4_4/props.plist	1969-12-31 19:00:00.000000000 -0500
@@ -5,7 +5,7 @@
 	<key>architecture</key>
 	<string>x86_64</string>
 	<key>homepage</key>
-	<string>http://fuse.sourceforge.net</string>
+	<string>https://github.com/libfuse/libfuse</string>
 	<key>installed_size</key>
 	<integer>333857</integer>
 	<key>license</key>
@@ -15,7 +15,7 @@
 	<key>pkgname</key>
 	<string>fuse</string>
 	<key>pkgver</key>
-	<string>fuse-2.9.4_3</string>
+	<string>fuse-2.9.4_4</string>
 	<key>run_depends</key>
 	<array>
 		<string>glibc&gt;=2.8_1</string>
@@ -34,6 +34,6 @@
 	<key>short_desc</key>
 	<string>Filesystem in Userspace</string>
 	<key>version</key>
-	<string>2.9.4_3</string>
+	<string>2.9.4_4</string>
 </dict>
 </plist>
```

Comment (stpx) [inline srcpkgs/fuse/template]: `fuse-devel`:
```diff
diff -Nur ./fuse-devel-2.9.4_3/files.plist ./fuse-devel-2.9.4_4/files.plist
--- ./fuse-devel-2.9.4_3/files.plist	1969-12-31 19:00:00.000000000 -0500
+++ ./fuse-devel-2.9.4_4/files.plist	1969-12-31 19:00:00.000000000 -0500
@@ -31,7 +31,7 @@
 			<key>file</key>
 			<string>/usr/lib/pkgconfig/fuse.pc</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>98506180c8804b973b3c2b58c0c829d525bd3630f7adf1d0ba32b566856b3510</string>
 		</dict>
@@ -39,23 +39,23 @@
 			<key>file</key>
 			<string>/usr/lib/libulockmgr.a</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
-			<string>0d1a8d5a927c0b8cce000292b34838218c3057a7a6a4f4114d47a4225384ba37</string>
+			<string>9445acda2d82adbabbbf18cb2b72ac923828a93da49d0cbf8c58de30b8341c4b</string>
 		</dict>
 		<dict>
 			<key>file</key>
 			<string>/usr/lib/libfuse.a</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701147</integer>
 			<key>sha256</key>
-			<string>870b789b27dc6ae52925ce5a8be426e2727432129df44f47f27fe3b076c86afd</string>
+			<string>220f5ee9ecb1ba5ff58cc1fec9ab399dc746602377289225886c4fcbfdfe69c5</string>
 		</dict>
 		<dict>
 			<key>file</key>
 			<string>/usr/include/ulockmgr.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>e970ad4661647c908efd892159104fcfe851e46da21140990b5048c704e1ec3b</string>
 		</dict>
@@ -63,7 +63,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>314a783a92f1be77450e7cdc798170fb4920b86e2ede3c1e4ff9214953532199</string>
 		</dict>
@@ -71,7 +71,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_compat.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>8d4e140eb962df31156e310064520e6f3663aa01c4b28a9df6d5a4792d793c7b</string>
 		</dict>
@@ -79,7 +79,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_lowlevel.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>eea84089a6baed11347121cbdcd63db1fa8ebb244e5813511c2564c26a707ac7</string>
 		</dict>
@@ -87,7 +87,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/cuse_lowlevel.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>5801a4f378ae0fbfdea6321159fa88805dc13799ac71ffba566b402c05afdefa</string>
 		</dict>
@@ -95,7 +95,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>df523fa6a7110d9bee78169c3e59510e16eae0c41df39a8823310f28730a67ad</string>
 		</dict>
@@ -103,7 +103,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_opt.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>ae678aebb063aba3112c9b2f62c013e7490daaf29ee4a41edf602eb6775bb563</string>
 		</dict>
@@ -111,7 +111,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_lowlevel_compat.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>a619f56de787f1da2d375fe22f08822f093134068492cdaca6a13d822212bc16</string>
 		</dict>
@@ -119,7 +119,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_common.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>5fb762f230ae60127fdcfc2d21e0ce43de28bfd050d2a1d1cf972f7222c36b1c</string>
 		</dict>
@@ -127,7 +127,7 @@
 			<key>file</key>
 			<string>/usr/include/fuse/fuse_common_compat.h</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>sha256</key>
 			<string>bbfcaff369604fc5f14eb92386f38f70c2ab7c6c9a4b28db022a386e69858a70</string>
 		</dict>
@@ -138,7 +138,7 @@
 			<key>file</key>
 			<string>/usr/lib/libulockmgr.so</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>target</key>
 			<string>/usr/lib/libulockmgr.so.1.0.1</string>
 		</dict>
@@ -146,7 +146,7 @@
 			<key>file</key>
 			<string>/usr/lib/libfuse.so</string>
 			<key>mtime</key>
-			<integer>1451698761</integer>
+			<integer>1451701146</integer>
 			<key>target</key>
 			<string>/usr/lib/libfuse.so.2.9.4</string>
 		</dict>
diff -Nur ./fuse-devel-2.9.4_3/props.plist ./fuse-devel-2.9.4_4/props.plist
--- ./fuse-devel-2.9.4_3/props.plist	1969-12-31 19:00:00.000000000 -0500
+++ ./fuse-devel-2.9.4_4/props.plist	1969-12-31 19:00:00.000000000 -0500
@@ -5,7 +5,7 @@
 	<key>architecture</key>
 	<string>x86_64</string>
 	<key>homepage</key>
-	<string>http://fuse.sourceforge.net</string>
+	<string>https://github.com/libfuse/libfuse</string>
 	<key>installed_size</key>
 	<integer>506791</integer>
 	<key>license</key>
@@ -15,14 +15,14 @@
 	<key>pkgname</key>
 	<string>fuse-devel</string>
 	<key>pkgver</key>
-	<string>fuse-devel-2.9.4_3</string>
+	<string>fuse-devel-2.9.4_4</string>
 	<key>run_depends</key>
 	<array>
-		<string>fuse&gt;=2.9.4_3</string>
+		<string>fuse&gt;=2.9.4_4</string>
 	</array>
 	<key>short_desc</key>
 	<string>Filesystem in Userspace - development files</string>
 	<key>version</key>
-	<string>2.9.4_3</string>
+	<string>2.9.4_4</string>
 </dict>
 </plist>
Binary files ./fuse-devel-2.9.4_3/usr/lib/libfuse.a and ./fuse-devel-2.9.4_4/usr/lib/libfuse.a differ
Binary files ./fuse-devel-2.9.4_3/usr/lib/libulockmgr.a and ./fuse-devel-2.9.4_4/usr/lib/libulockmgr.a differ
```

File: srcpkgs/fuse-sshfs/template (hunk 1 above)
Comment (stpx) [inline srcpkgs/fuse-sshfs/template]: `fuse-sshfs`:
```diff
diff -Nur ./fuse-sshfs-2.5_4/files.plist ./fuse-sshfs-2.5_5/files.plist
--- ./fuse-sshfs-2.5_4/files.plist	2015-07-24 05:19:11.000000000 -0400
+++ ./fuse-sshfs-2.5_5/files.plist	1969-12-31 19:00:00.000000000 -0500
@@ -31,15 +31,15 @@
 			<key>file</key>
 			<string>/usr/bin/sshfs</string>
 			<key>mtime</key>
-			<integer>1437729551</integer>
+			<integer>1451931607</integer>
 			<key>sha256</key>
-			<string>190ae4b3e66a58e39be69306dae52e8936a0844ddeb561642c1694e3caa93a49</string>
+			<string>1fb30f24acab79597a6d648be0265df1c5a64a5d82fc4de1f12cf874080ff314</string>
 		</dict>
 		<dict>
 			<key>file</key>
 			<string>/usr/share/man/man1/sshfs.1</string>
 			<key>mtime</key>
-			<integer>1437729551</integer>
+			<integer>1451931607</integer>
 			<key>sha256</key>
 			<string>2f669bdd00a90713e17de601508f74d17ce629fb01f4a0cdbc71aef5c173f0df</string>
 		</dict>
diff -Nur ./fuse-sshfs-2.5_4/props.plist ./fuse-sshfs-2.5_5/props.plist
--- ./fuse-sshfs-2.5_4/props.plist	2015-07-24 05:19:11.000000000 -0400
+++ ./fuse-sshfs-2.5_5/props.plist	1969-12-31 19:00:00.000000000 -0500
@@ -4,22 +4,18 @@
 <dict>
 	<key>architecture</key>
 	<string>x86_64</string>
-	<key>build-date</key>
-	<string>2015-07-24 09:19 UTC</string>
 	<key>homepage</key>
-	<string>http://fuse.sourceforge.net/sshfs.html</string>
+	<string>https://github.com/libfuse/sshfs</string>
 	<key>installed_size</key>
-	<integer>88578</integer>
+	<integer>88490</integer>
 	<key>license</key>
 	<string>GPL-2</string>
 	<key>maintainer</key>
 	<string>Juan RP &lt;xtraeme@voidlinux.eu&gt;</string>
-	<key>packaged-with</key>
-	<string>xbps-src-113</string>
 	<key>pkgname</key>
 	<string>fuse-sshfs</string>
 	<key>pkgver</key>
-	<string>fuse-sshfs-2.5_4</string>
+	<string>fuse-sshfs-2.5_5</string>
 	<key>run_depends</key>
 	<array>
 		<string>openssh&gt;=0</string>
@@ -37,9 +33,7 @@
 	</array>
 	<key>short_desc</key>
 	<string>FUSE client based on the SSH File Transfer Protocol</string>
-	<key>source-revisions</key>
-	<string>fuse-sshfs:f10a966</string>
 	<key>version</key>
-	<string>2.5_4</string>
+	<string>2.5_5</string>
 </dict>
 </plist>
Binary files ./fuse-sshfs-2.5_4/usr/bin/sshfs and ./fuse-sshfs-2.5_5/usr/bin/sshfs differ
```

File: srcpkgs/python-fuse/template (hunk 3 above)
Comment (stpx) [inline srcpkgs/python-fuse/template]: `python-fuse`:
```diff
diff -Nur ./python-fuse-0.2.1_1/files.plist ./python-fuse-0.2.1_2/files.plist
--- ./python-fuse-0.2.1_1/files.plist	2014-08-16 11:48:44.000000000 -0400
+++ ./python-fuse-0.2.1_2/files.plist	1969-12-31 19:00:00.000000000 -0500
@@ -29,39 +29,51 @@
 	<array>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuseparts/setcompatwrap.py</string>
+			<string>/usr/lib/python2.7/site-packages/fuse.py</string>
+			<key>mtime</key>
+			<integer>1271282972</integer>
 			<key>sha256</key>
-			<string>51a98209f233f3b2a36004681591b3e46b41a8179bc25fd48ea3807cbcd2ca50</string>
+			<string>7783d276091be5f3272898f9fe6b440ca14d31a26e46dc3cec24f1bbfb531388</string>
 		</dict>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuseparts/subbedopts.py</string>
+			<string>/usr/lib/python2.7/site-packages/fuseparts/_fusemodule.so</string>
+			<key>mtime</key>
+			<integer>1451932299</integer>
 			<key>sha256</key>
-			<string>83335600507afdc05e0119c8991a803150e3df4139048e5d7bc4636a25d2c5d5</string>
+			<string>9ebefef087bd4c924aafd5605a0e852a6636a81b4001151ed69fa032ed5676fa</string>
 		</dict>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuseparts/__init__.py</string>
+			<string>/usr/lib/python2.7/site-packages/fuseparts/subbedopts.py</string>
+			<key>mtime</key>
+			<integer>1271282972</integer>
 			<key>sha256</key>
-			<string>1df8d53aba539e667ec551426125665f9d045da050789b5e5071be3c3ea242cf</string>
+			<string>83335600507afdc05e0119c8991a803150e3df4139048e5d7bc4636a25d2c5d5</string>
 		</dict>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuseparts/_fusemodule.so</string>
+			<string>/usr/lib/python2.7/site-packages/fuseparts/setcompatwrap.py</string>
+			<key>mtime</key>
+			<integer>1271282972</integer>
 			<key>sha256</key>
-			<string>3fd70097faaba2879b40c5c94b1f75ea2eb306e1ad149951d6902ad2dd139185</string>
+			<string>51a98209f233f3b2a36004681591b3e46b41a8179bc25fd48ea3807cbcd2ca50</string>
 		</dict>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuse_python-0.2.1-py2.7.egg-info</string>
+			<string>/usr/lib/python2.7/site-packages/fuseparts/__init__.py</string>
+			<key>mtime</key>
+			<integer>1271282972</integer>
 			<key>sha256</key>
-			<string>0bdd519ba2ecac9c06b53dd154ba7f953d8aacc027c5919aec8876f2835831dd</string>
+			<string>1df8d53aba539e667ec551426125665f9d045da050789b5e5071be3c3ea242cf</string>
 		</dict>
 		<dict>
 			<key>file</key>
-			<string>/usr/lib/python2.7/site-packages/fuse.py</string>
+			<string>/usr/lib/python2.7/site-packages/fuse_python-0.2.1-py2.7.egg-info</string>
+			<key>mtime</key>
+			<integer>1451932299</integer>
 			<key>sha256</key>
-			<string>7783d276091be5f3272898f9fe6b440ca14d31a26e46dc3cec24f1bbfb531388</string>
+			<string>0bdd519ba2ecac9c06b53dd154ba7f953d8aacc027c5919aec8876f2835831dd</string>
 		</dict>
 	</array>
 </dict>
diff -Nur ./python-fuse-0.2.1_1/props.plist ./python-fuse-0.2.1_2/props.plist
--- ./python-fuse-0.2.1_1/props.plist	2014-08-16 11:48:44.000000000 -0400
+++ ./python-fuse-0.2.1_2/props.plist	1969-12-31 19:00:00.000000000 -0500
@@ -4,22 +4,18 @@
 <dict>
 	<key>architecture</key>
 	<string>x86_64</string>
-	<key>build-date</key>
-	<string>2014-08-16 17:48 CEST</string>
 	<key>homepage</key>
-	<string>http://sourceforge.net/projects/fuse/</string>
+	<string>https://github.com/libfuse/python-fuse</string>
 	<key>installed_size</key>
-	<integer>80470</integer>
+	<integer>82606</integer>
 	<key>license</key>
 	<string>LGPL-2.1</string>
 	<key>maintainer</key>
 	<string>Eivind Uggedal &lt;eivind@uggedal.com&gt;</string>
-	<key>packaged-with</key>
-	<string>xbps-src-113</string>
 	<key>pkgname</key>
 	<string>python-fuse</string>
 	<key>pkgver</key>
-	<string>python-fuse-0.2.1_1</string>
+	<string>python-fuse-0.2.1_2</string>
 	<key>run_depends</key>
 	<array>
 		<string>fuse&gt;=2.8.1_1</string>
@@ -35,9 +31,7 @@
 	</array>
 	<key>short_desc</key>
 	<string>FUSE bindings for Python2</string>
-	<key>source-revisions</key>
-	<string>python-fuse:1aa411c</string>
 	<key>version</key>
-	<string>0.2.1_1</string>
+	<string>0.2.1_2</string>
 </dict>
 </plist>
Binary files ./python-fuse-0.2.1_1/usr/lib/python2.7/site-packages/fuseparts/_fusemodule.so and ./python-fuse-0.2.1_2/usr/lib/python2.7/site-packages/fuseparts/_fusemodule.so differ
```

**Comment by stpx** [comment]
@xtraeme Ok to merge?

FTR, The project moved to github and the respective `distfiles` links are dead.
```sh
$ ./xbps-src pkg fuse-sshfs    
[*] Updating `http://repo.voidlinux.eu/current/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/nonfree/x86_64-repodata' ...
[*] Updating `http://muslrepo.voidlinux.eu/current/x86_64-repodata' ...
ERROR: [reposync] failed to fetch file `http://muslrepo.voidlinux.eu/current/x86_64-repodata': Not Found
[*] Updating `http://muslrepo.voidlinux.eu/current/nonfree/x86_64-repodata' ...
ERROR: [reposync] failed to fetch file `http://muslrepo.voidlinux.eu/current/nonfree/x86_64-repodata': Not Found
[*] Updating `http://repo.voidlinux.eu/current/multilib/x86_64-repodata' ...
[*] Updating `http://repo.voidlinux.eu/current/multilib/nonfree/x86_64-repodata' ...
=> fuse-sshfs-2.5_4: removing autodeps, please wait...
=> fuse-sshfs-2.5_4: building ...
   [host] pkg-config-0.29_1: found (http://repo.voidlinux.eu/current)
   [target] libglib-devel-2.46.2_1: found (http://repo.voidlinux.eu/current)
   [target] fuse-devel-2.9.4_3: found (http://repo.voidlinux.eu/current)
   [runtime] openssh>=0: found (http://repo.voidlinux.eu/current)
=> fuse-sshfs-2.5_4: installing host dependency 'pkg-config-0.29_1' ...
=> fuse-sshfs-2.5_4: installing target dependency 'libglib-devel-2.46.2_1' ...
=> fuse-sshfs-2.5_4: installing target dependency 'fuse-devel-2.9.4_3' ...
=> fuse-sshfs-2.5_4: running do-fetch hook: 00-distfiles ...
=> fuse-sshfs-2.5_4: fetching distfile 'sshfs-fuse-2.5.tar.gz'...
looking up downloads.sourceforge.net
connecting to downloads.sourceforge.net:80
requesting http://downloads.sourceforge.net/sourceforge/fuse/sshfs-fuse-2.5.tar.gz
http://downloads.sourceforge.net/sourceforge/fuse/sshfs-fuse-2.5.tar.gz: Not Found
=> ERROR: fuse-sshfs-2.5_4: failed to fetch sshfs-fuse-2.5.tar.gz.
```

**Pull request merged**